madgiraffestudios/src/junkcode.txt

```diff
@@ -1 +1,64 @@
-text-center pt-5 mx-auto w-50
\ No newline at end of file
+text-center pt-5 mx-auto w-50
```

Comment:
@MrGDS2 remove this junkcode file as well

Reply:
will resolve in the next PR